pkg/distributor/distributor.go

```diff
@@ -41,18 +51,55 @@ var (
 	rfStats           = usagestats.NewInt("distributor_replication_factor")
 )
 
+type ShardStreamMode string
+
+const (
+	NeverShardMode       ShardStreamMode = "never"
+	AlwaysShardMode                      = "always"
+	DynamicallyShardMode                 = "dynamic"
+)
+
+// TODO: document shard streams configurations/modes.
```

Comment:
Just making sure that these are addressed before the merge :slightly_smiling_face:

Reply:
In the PR description there's an explanation of why we're not documenting this yet.

Reply:
:+1:  done today